Makefile

```diff
@@ -123,21 +123,21 @@ install: install-server install-ui ## Build and install
 watch: ## Watch packages. For use alongside jupyter lab --watch
 	export PATH=$$(pwd)/node_modules/.bin:$$PATH && lerna run watch --parallel
 
-test-server: install-server ## Run unit tests
+test-server: install-server # Run unit tests
 	pytest -v elyra
 
-test-ui: lint-ui test-ui-unit test-ui-integration ## Run frontend tests
+test-ui: lint-ui test-ui-unit test-ui-integration # Run frontend tests
 
-test-ui-integration: ## Run frontend cypress integration tests
-	npm run test:integration
-
-test-ui-unit: ## Run frontend jest unit tests
+test-ui-unit: # Run frontend jest unit tests
 	npm run test:unit
 
-test-ui-debug: ## Open cypress integration test debugger
+test-ui-integration: # Run frontend cypress integration tests
+	npm run test:integration
+
+test-ui-integration-debug: # Open cypress integration test debugger
 	npm run test:integration:debug
```

Comment:
Shouldn't it be `test-integration` / `test-integration-debug`, because it tests the system as a whole, not just the UI?

Reply:
Maybe, but as there is a possibility for us to have backend integration tests as well, and there is already a pattern of using UI for UI-related targets, I would prefer to leave it this way for now.

Reply:
hmm - I figured _integration_ meant front and back already.  What does integration mean within the front-end?

Reply:
I think the what is considered an `integration` test can be a bit of a blurred line, so I tend to not use the term. I normally opt for calling cypress tests that cover the entire system `end-to-end` tests. Then all of my ui tests are a mix `unit` and what could be considered `integration` tests (excluding the E2E tests), but I don't really make any kind of distinction between them and just call them "tests"/ "ui tests" / "unit tests"

If we want to be able to use "integration tests" for things that are exclusively frontend/backend, maybe we should also opt for a different word for tests that test the system as a whole?